### ISSUE TITLE
Cookie Clear Link

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface#286]
I just had an issue with the Washington Post on my machine, and they did something that gave me an idea for support.

Support determined there was a cookie problem on my machine, and rather than just asking me to clear cookies, they asked me to go to a link:

![image](https://user-images.githubusercontent.com/4837554/107777956-7fbe9480-6d11-11eb-9674-33256a42473d.png)

This makes it very easy for the end-user who doesn't know how to clear cookies.  Considering we are likely to run into cookie issues during the beta, and maybe even in times in the future, that would be a good thing to add to our support arsenal.

Note that their (javascript-based) cookie clean selectively clears only the WPO cookies.  We should do the same for PEP, and use a similar URL pattern.

**Comment by nrshapiro** [comment]
What is the link for this?  Perhaps we should we, at least for the beta, put it on the ? menu under Support?  We may want to remove it later, but I'm afraid they're going to need it for a bit.  I just had a problem on Production where no matter how many times I tried to log out, the menu just still said "logout" (see video below).  That was the build on production yesterday, from the 15, and after that experience, I just updated it to the build from the 16.

https://user-images.githubusercontent.com/4837554/111468216-af3f3300-86fb-11eb-8529-9c645607d62e.mp4
